als-server/shared/src/test/scala/org/mulesoft/als/server/textsync/DialectRegistryTest.scala

```diff
@@ -32,8 +34,7 @@ class DialectRegistryTest extends LanguageServerBaseTest {
      factory.workspaceManager)
   }
 
-  // todo: enable test after APIMF-3305 is adopted
-  ignore("Remove old dialect after modifying it") {
+  test("Remove old dialect after modifying it") {
```

Comment:
is this the only test? maybe add some more, changing dialects, version/name, properties, etc
asserting there are no duplicated dialects registered and that the last one is the registered at the end